### DISCUSSION TITLE
Move from Alpine to Wolfi

### DISCUSSION THREAD

**Discussion by sando38** [nextcloud/all-in-one#4762] [Ideas] [answered]
Hello @szaimen , the current container images are build based on Alpine Linux. We maintain container images for [ejabberd](https://github.com/processone/ejabberd) and [eturnal](https://github.com/processone/eturnal) currently also based on Alpine linux, however, we noticed great differences in performance and resource consumption between musl-libc and glibc.

We are not aware of the impact on the nextcloud ecosystem, but it may be worth to check https://github.com/wolfi-dev/os , which is in its nature pretty "Alpine" but based on glibc.

For the future outlook, we may also switch our official images to be based on wolfi-os instead of Alpine, but that is nothing we are doing in the next 3 months.. and may keep a Alpine variant coexisting. ;)

Cheers, Saarko

**Comment by szaimen** [comment, marked as answer]
Hi Saarko :) thanks for the idea! However for most images that are performance-relevant, we use pre-built official ones so I dont see a good and maintainable way for us to switch to wolfi without having to re-implement a lot of the images which is not something I would like to do honestly.
